flink-staging/flink-streaming/flink-streaming-core/src/main/java/org/apache/flink/streaming/api/state/EagerStateStore.java

```diff
@@ -43,7 +44,7 @@ public EagerStateStore(StateCheckpointer<S, C> checkpointer, StateHandleProvider
 	}
 
 	@Override
-	public S getStateForKey(Serializable key) throws Exception {
+	public S getStateForKey(Serializable key) throws IOException {
```

Comment:
While this is currently not used, once we implement things like out-of-core state access or lazy state access this will probably become necessary.